examples/amp-consent/geooverride-iframe.amp.html

```diff
@@ -187,30 +187,27 @@ <h3>Image that is NOT blocked by consent</h3>
   <amp-consent id='ABC' layout='nodisplay'>
     <script type="application/json">{
       "consentInstanceId": "geotest",
-      "promptUI": "ui1",
+      "promptUISrc": "/examples/amp-consent/prompt.html",
```

Comment:
hmmm can we have another example to use the `<amp-iframe>`. Using `<amp-iframe>` and `promptUISrc` are two very different approaches.